week2/Task3/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿// 3. Adding time
+// 4. Adding minutes
+
+var time = new JupiterTime(2, 20);
+var timeIn1Hour = time.AddHours(1);
+var time1 = new JupiterTime(1, 21);
+var timeIn20Minutes = time1.AddMinutes(20);
+
+PrintTime(timeIn1Hour);
+PrintTime(timeIn20Minutes);
+
+void PrintTime(string time)
+{
+    Console.WriteLine($"{time}");
+} 
+public class JupiterTime
+{
+    public int Hours { get; set; }
+    public int Minutes { get; set; } 
+    public JupiterTime(int hours, int minutes){
+        Hours = hours;
+        Minutes=minutes;
+
+        if(Minutes>59){
+            Hours += minutes/60;
+            Minutes = minutes % 60;
+        }
+    }
+    public string AddHours(int hour){
+         Hours +=hour;
+         return ($"{Hours}:{Minutes}");
+    }
+    public string AddMinutes(int minute){
+         Minutes +=minute;
+         return ($"{Hours}:{Minutes}");
```

Comment:
These two methods should return a new object, so it can be something like: return new JupiterTime(Hours, Minutes)
Then the return type of the methods would change from string to JupiterTime.

I can expand on the explanation if you would like, just let me know 😊